packages/dotnet-sdk/src/TeamsFx/Configuration/AuthenticationOption.cs

```diff
@@ -27,9 +27,37 @@ public class AuthenticationOptions
     [Required(ErrorMessage = "Client secret is required")]
     public string ClientSecret { get; set; }
 
+    /// <summary>
+    /// AAD tenant id.
+    /// </summary>
+    [RegularExpression(@"^[0-9A-Fa-f\-]{36}$")]
+    public string TenantId { get; set; }
+
+    /// <summary>
+    /// Application ID URI.
+    /// </summary>
+    public string ApplicationIdUri { get; set; }
+
     /// <summary>
     /// Authority URL that is used in OAuth On-behalf-of flow.
     /// </summary>
     [RegularExpression(@"^http(s)?://[-a-zA-Z0-9@:%._\+~#=/]{1,100}$")]
     public string OAuthAuthority { get; set; }
```

Comment:
I saw it's OAuthAuthority not OAuthAuthority host, does it mean it will contain tenant id, and we do not need to ask for another TenantId config?

Reply:
Fixed. Will extract tenant Id from OAuthAuthoirity.